stories/adminbar.stories.js

```diff
@@ -21,57 +21,905 @@
  */
 import { storiesOf } from '@storybook/react';
 
-/**
- * WordPress dependencies
- */
-import { addFilter, removeAllFilters } from '@wordpress/hooks';
-
 /**
  * Internal dependencies
  */
 import { GoogleSitekitAdminbar } from '../assets/js/googlesitekit-adminbar';
 import { googlesitekit as wpAdminBarData } from '../.storybook/data/blog---googlesitekit';
-import AnalyticsAdminbarWidget from '../assets/js/modules/analytics/components/adminbar/AnalyticsAdminbarWidget';
-import GoogleSitekitSearchConsoleAdminbarWidget from '../assets/js/modules/search-console/components/adminbar/GoogleSitekitSearchConsoleAdminbarWidget';
-import { createAddToFilter } from '../assets/js/util/helpers';
-import { CORE_SITE } from '../assets/js/googlesitekit/datastore/site/constants';
 import { CORE_USER } from '../assets/js/googlesitekit/datastore/user/constants';
-import { WithTestRegistry } from '../tests/js/utils';
-import CollectModuleData from '../assets/js/components/data/collect-module-data';
+import { provideModules, provideSiteInfo, WithTestRegistry } from '../tests/js/utils';
+import { STORE_NAME as MODULES_SEARCH_CONSOLE } from '../assets/js/modules/search-console/datastore/constants';
+import { STORE_NAME as MODULES_ANALYTICS } from '../assets/js/modules/analytics/datastore/constants';
 
 storiesOf( 'Global', module )
 	.add( 'Admin Bar', () => {
 		global._googlesitekitLegacyData = wpAdminBarData;
 
-		const setupRegistry = ( { dispatch } ) => {
-			dispatch( CORE_SITE ).receiveSiteInfo( {
-				usingProxy: true,
-				referenceSiteURL: 'https://example.com',
-				adminURL: 'https://example.com/wp-admin/',
-				siteName: 'My Site Name',
+		const setupRegistry = ( registry ) => {
+			// Set the Story site information.
+			provideSiteInfo( registry, {
 				currentEntityURL: 'https://www.sitekitbygoogle.com/blog/',
 				currentEntityTitle: 'Blog test post for Google Site Kit',
-				currentEntityType: 'blog',
-				currentEntityID: 2,
-			} );
-			dispatch( CORE_USER ).receiveGetAuthentication( {
-				authenticated: true,
-				requiredScopes: [],
-				grantedScopes: [],
 			} );
-		};
 
-		const addGoogleSitekitSearchConsoleAdminbarWidget = createAddToFilter( <GoogleSitekitSearchConsoleAdminbarWidget /> );
-		const addAnalyticsAdminbarWidget = createAddToFilter( <AnalyticsAdminbarWidget /> );
+			// Set up the search console and analytics modules stores but provide no data.
+			provideModules( registry, [
+				{ slug: 'search-console', active: true, connected: true },
+				{ slug: 'analytics', active: true, connected: true },
+			] );
+
+			registry.dispatch( CORE_USER ).setReferenceDate( '2021-01-28' );
+
+			// Search Console mock data
+			const searchConsoleData = [
```

Comment:
This is a _lot_ of data to have in the story itself, let's move this outside the file to a fixture file or something to improve readability of this file.

Reply:
Moved to fixtures.